app/views/dependencies/_dependencies.html.erb

```diff
@@ -0,0 +1,31 @@
+<% if @dependencies.key?(scope) %>
+  <% if scope == "development" %>
+    <div id="<%=params[:rubygem_id]+scope%>"><span class="scope"><%= scope.titlecase %> :</span>
```

Comment:
Using `params` (user input) directly is very dangerous, please use `@latest_version.rubygem.name`.